build/tasks/build.js

```diff
@@ -65,7 +65,7 @@ module.exports = function( grunt ) {
 		if ( /.\/var\//.test( path.replace( process.cwd(), "" ) ) ) {
 			contents = contents
 				.replace(
-					/define\([\w\W]*?return/,
+					/define\(\s*(["'])[\w\W]*?\1[\w\W]*?return/,
```

Comment:
Simpler suggestion: https://jsbin.com/wozinogigo/1/edit?js,console,output
```suggestion
					/define\([\w\W]*?\breturn\b/,
```

Reply:
Technically, there could be a directory named `return`. Not the best idea, perhaps, but my regex should work for all of those weird cases.